Library/Homebrew/diagnostic.rb

```diff
@@ -513,7 +514,7 @@ def check_user_path_3
           Homebrew's sbin was not found in your PATH but you have installed
           formulae that put executables in #{HOMEBREW_PREFIX}/sbin.
           Consider setting the PATH for example like so
-            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{shell_profile}
+            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{Utils::Shell.shell_profile}
```

Comment:
I'm really not sure what to do here, since this is a user-facing message. Were you thinking of detecting the shell and sending an appropriate message?

In that case, I think the fish message should show something like this for `fish`.

    fish: 
      1) universal variable.
      set -U fish_user_paths #{HOMEBREW_PREFIX}/sbin $fish_user_paths

      2) modify /.config/fish/fish.config
      echo 'set -x PATH "#{HOMEBREW_PREFIX}"/sbin $PATH' >> #{Utils::Shell.shell_profile}

From the fish tutorial: http://fishshell.com/docs/current/tutorial.html#tut_path

> A faster way is to modify the $fish_user_paths universal variable, which is automatically prepended to $PATH. For example, to permanently add /usr/local/bin to your $PATH, you could write:

`tcsh` would just be 

    tcsh:
        set path = ( #{HOMEBREW_PREFIX}/sbin $path:q )